src/app/components/Embeds/EmbedImages/index.tsx

```diff
@@ -19,8 +19,10 @@ type Props = {
 };
 
 const EmbedImages = ({ blocks: embedImages }: PropsWithChildren<Props>) => {
-  const { env } = useContext(RequestContext);
-  const image = embedImages?.slice(-1)?.[0]?.model?.blocks;
+  const { isAmp, env } = useContext(RequestContext);
+  const ampImage = embedImages?.[1]?.model?.blocks;
+  const canonicalImage = embedImages?.[2]?.model?.blocks;
```

Comment:
I haven't been across the data structure too much, but is there a difference in the images to support AMP and Canonical?

Reply:
Yeah, so when looking into the work, it looks like the CPS implementation uses the 860px view port image for canonical and the 640px one for AMP, there's a separate ticket to look into responsiveness with srcset https://jira.dev.bbc.co.uk/browse/WSTEAM1-536